core/src/main/java/org/apache/accumulo/core/client/impl/DelegationToken.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.accumulo.core.client.security.tokens;
+package org.apache.accumulo.core.client.impl;
```

Comment:
My thinking on the change was that DelegationToken needed to be changed to an opaque type.  While thinking about, just returning AuthenticationToken in the API seemed like the best way to make it opaque in the API.  If it needs additional public methods, or you think it may in the future we could have a more specialized opaque type for delegation token that simply extends AuthenticationToken.